lang/en/package.html

```diff
@@ -17,66 +17,61 @@ <h2 class="details-main--title d-inline-block m-2">…</h2>
             <!-- react-text: 8 -->
             <!-- /react-text -->
           </a><span class="ais-Hit--popular null" title="0 downloads in the last 30 days">0</span>
-          <!-- react-empty: 10 --><span class="ais-Hit--version">0.0.0</span></div>
+          <!-- react-empty: 10 --><span class="ais-Hit--version"></span></div>
```

Comment:
I think it'd be nice to have a script. Then we could include it as part of the build, ensuring that it's always up-to-date (I'm not a fan of committing build artifacts to source control).